modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Suites/SuiteForm.tsx

```diff
@@ -75,7 +80,46 @@ const SuiteForm = () => {
 	});
 
 	const smartSuite = watch('smartSuite');
-	const caseParameters = watch('caseParameters');
+	const caseParametersWath = watch('caseParameters');
+
+	const useCaseFilter = () => {
```

Comment:
```suggestion
	const getCaseParametersFilter = useCallback... / useMemo
```

if callback = getCaseParametersFilter()
if memo = caseParametersFilter